backend/company_api.py

```
@@ -1028,6 +1028,296 @@ async def cancel_onboarding(
+            messages=[
+                {"role": "system", "content": "You return only valid JSON arrays. No explanation."},
+                {"role": "user", "content": prompt},
+            ],
+            temperature=0.4,
+        )
+
+        # Extract JSON from the response (handle markdown fences)
+        import re
+        json_match = re.search(r"\[.*\]", raw.strip(), re.DOTALL)
+        if json_match:
+            questions = json.loads(json_match.group())
```

Ruff (0.15.15)

[error] Undefined name `json`

(F821)